market_nextdoor_api/urls.py

```diff
@@ -16,24 +16,24 @@
 """
 from django.contrib import admin
 from django.urls import path
-from market_nextdoor_api import views
+from .views import market_views, customer_views, vendor_views, item_views, customer_views, preorder_views
 from rest_framework.urlpatterns import format_suffix_patterns
```

Comment:
Great implementation of what Antoine mentioned. This looks great, and glad to know we can import so many things this way!